tests/test_ptk_multiline.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Tests sample inputs to PTK multiline and checks parser response"""
+import nose
+from nose.tools import assert_equal
+from unittest.mock import MagicMock, patch
+
+import builtins
+from xonsh.environ import Env
+builtins.__xonsh_env__ = Env()
+builtins.__xonsh_env__['INDENT'] = '    '
```

Comment:
These lines could have side effects for other tests.  probably best to stick all of this stuff in a setup() function and then unwind it all in a teardown(). Same goes for below as well.